### DISCUSSION TITLE
Lock days

### DISCUSSION THREAD

**Discussion by nathanmedhurstschoolshr** [easepick/easepick#51] [General]
Hi!

Recently started using this and having a slight issue with the LockPlugin.

Im looking to remove weekends from all dates and i can see some heavy documentation on LitePicker to achieve this.

Any help?

**Comment by wakirin** [comment]
https://jsfiddle.net/waki/scrvpueg/

**Comment by nathanmedhurstschoolshr** [comment]
Thanks,

Thats really useful!

Trying to integrate it with the Range Plugin so we can use this for booking Holidays.

This is what i have so far: https://jsfiddle.net/nzLcktv4/

i need the Max Days to only be used against bookable days, is this something that is possible currently?

**Reply by wakirin** [reply]
Yes, possible.
Look for similar discussions to see how to implement this.

**Reply by nathanmedhurstschoolshr** [reply]
Thanks, will continue to investigate

**Reply by nathanmedhurstschoolshr** [reply]
Ive tried for a while to get this working but have got stuck on how, any support?
